public/js/controllers/game.js

```diff
@@ -246,47 +273,44 @@ angular.module('mean.system')
     const regex = /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,63}$/i;
     if (regex.test($scope.email) && !$scope.inviteList.includes($scope.email)) {
       if ($scope.inviteList.length > game.playerMaxLimit) {
-        $scope.animationsEnabled = true;
-        $scope.errorBody = `You have exceeded ${game.playerMaxLimit}, the  maximum amount of invite for this game.`;
-        $scope.modalInstance = $modal.open({
-          animation: $scope.animationsEnabled,
-          ariaLabelledBy: 'modal-title',
-          scope: $scope,
-          ariaDescribedBy: 'modal-body',
-          templateUrl: '/views/popup.html',
-          controller: 'GameController',
-          size: 'sm',
-          appendTo: angular.element(document).find('#gameplay-container'),
-          resolve: {
-            items: function () {
-              return $scope.errorBody;
-            }
-          }
-        });
-        $scope.modalInstance.result.then(function (selectedItem) {
-          $scope.select = selectedItem;
-        }, function () {
-          // Do nothing
-        });
-        return;
+        let message = `You have exceeded ${game.playerMaxLimit}, the  maximum amount of invite for this game.`;
+        let templateUrl = '/views/popup.html';
+        showModal(message, templateUrl);
       }
-      $http({
-        method: 'POST',
-        url: '/api/invite/user',
-        headers: { 'Content-Type': 'application/json' },
-        data: {
-          email: $scope.email,
-          link: document.URL,
-          sender: window.user.name
-        }
-      })
+      const link = '/api/invite/user';
```

Comment:
The reason I defined the link there was because I am defining different links for a different route. The first is the email invite and the second is the in-app invite which would be defined later.